public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -87,8 +88,8 @@ export class TableContainer extends PureComponent<Props> {
     return (
       <>
         {frames && frames.length === 0 && (
-          <PanelChrome title={'Table'} width={width} height={200}>
-            {() => <MetaInfoText metaItems={[{ value: '0 series returned' }]} />}
+          <PanelChrome title={t('explore.table.title', 'Table')} width={width} height={200}>
+            {() => <MetaInfoText metaItems={[{ value: t('explore.table.meta', '0 series returned')}]} />}
```

Comment:
We can probably call it `explore.table.no-data`

Reply:
@ifrost  hello I updated the name and solve the merge conflict